iceoryx_dds/include/iceoryx_dds/internal/gateway/iox_to_dds.inl

```diff
@@ -70,7 +70,7 @@ inline void Iceoryx2DDSGateway<channel_t, gateway_t>::discover(const capro::Capr
     {
         return;
     }
-    if (msg.m_subType == capro::CaproMessageSubType::SERVICE)
```

Comment:
You know the drill ... copyright year

Reply:
In all files please ... how much I dislike this ...

Reply:
hehe